deploy/docker/x86_64/Dockerfile

```
@@ -13,17 +13,38 @@
 # limitations under the License.
 
 # Modified from https://github.com/rootfs/nfs-ganesha-docker by Huamin Chen
-FROM fedora:30 AS build
+ARG FEDORA_VERSION=34
```

While updating this as part of this PR anyhow, one may consider going for 35 instead of 34.

https://hub.docker.com/_/fedora?tab=tags&page=1&ordering=last_updated

```suggestion
ARG FEDORA_VERSION=35
```

---

Oh, I see now that V3.4 was used in the root folder /Dockerfile